### PR TITLE
Prepare kubekins-e2e and kubekins-test images for 1.12 release

### PR DESCRIPTION
**Pull request by amwat**
Add 1.12 tag to kubekins-e2e.
Add make rule for kubekins-e2e 1.12.

Add Dockerfile-1.12 for kubekins-test.
Bump kubekins-test.
Update kubekins-test tag kubernetes verify scenario for 1.12

/cc @cjwagner @krzyzacy

### CODE REVIEW

**Comment by amwat** [comment]
/hold
Hold changes until the release.

File: scenarios/kubernetes_verify.py [hunk 1]
```diff
@@ -31,8 +31,9 @@
     '1.8': '1.8-v20170906-3a1c5ae5',
```
Comment (krzyzacy) [inline scenarios/kubernetes_verify.py]: let's get rid of 1.7 and 1.8, and their DOCKERFILEs as well

**Comment by krzyzacy** [comment]
push the image and let's get this one in first?

**Comment by amwat** [comment]
pushed

**Pull request merged**